plugin/pkg/admission/limitranger/admission.go

```diff
@@ -68,7 +86,21 @@ func (l *limitRanger) Admit(a admission.Attributes) (err error) {
 
 // NewLimitRanger returns an object that enforces limits based on the supplied limit function
 func NewLimitRanger(client client.Interface, limitFunc LimitFunc) admission.Interface {
-	return &limitRanger{client: client, limitFunc: limitFunc}
+	indexer := cache.NewIndexer(cache.MetaNamespaceKeyFunc, cache.Indexers{"namespace": cache.MetaNamespaceIndexFunc})
```

Comment:
Would it make sense to move this and the similar code below to a helper function in pkg/client/cache called `NewNamespaceKeyedIndexerAndReflector(lw ListWatcher)`